src/test/scala/dpla/ingestion3/enrichments/StringUtilsTest.scala

```diff
@@ -204,13 +204,42 @@ class StringUtilsTest extends FlatSpec with BeforeAndAfter {
     assert(enrichedValue === "")
   }
 
-  "findAndRemoveAll" should "remove all stop words from the given string" in {
-    val stopWords = Set("jp2", "application/xml")
+  "stripInvalidFormats" should "remove all stop words from the given string" in {
     val originalValue = "application/xml photograph   jp2"
-    val enrichedValue = originalValue.findAndRemoveAll(stopWords)
+    val enrichedValue = originalValue.stripInvalidFormats
     assert(enrichedValue === "photograph")
   }
-  
+  it should "not case about case" in {
+    val originalValue = "application/XML Photograph   jp2"
+    val enrichedValue = originalValue.stripInvalidFormats
+    assert(enrichedValue === "Photograph")
+  }
+  it should "remove stop words that contain white space e.g. 'JPEG 2000'" in {
+    val originalValue = "JPEG 2000 Photograph"
+    val enrichedValue = originalValue.stripInvalidFormats
+    assert(enrichedValue === "Photograph")
+  }
+  it should "leave the original value alone if it contains no stopwords" in {
+    val originalValue = "Photograph"
+    val enrichedValue = originalValue.stripInvalidFormats
+    assert(enrichedValue === "Photograph")
+  }
+  it should "return empty string if format only contains stop words" in {
+    val originalValue = "  text/pdf tif  video/jpeg  \t video/jpeg2000"
+    val enrichedValue = originalValue.stripInvalidFormats
+    assert(enrichedValue === "")
+  }
+  it should "remove invalid values that contain reserved regex chars" in {
+    val originalValue = "3gpdash-qoe-report+xml photograph  "
+    val enrichedValue = originalValue.stripInvalidFormats
+    assert(enrichedValue === "photograph")
+  }
+  it should "remove a stop word if it exists within another term" in {
+    val originalValue = "application/xmlphotograph"
+    val enrichedValue = originalValue.stripInvalidFormats
+    assert(enrichedValue === "photograph")
+  }
+
```

Comment:
Just a quick sanity check - is it okay that some of the tests return `Photograph` (uppercase `P`) and others return `photograph` (lowercase `p`)?  I assume any upcasing or downcasing enrichments are handled elsewhere?

Reply:
that is a sanity check that the enrichment isn't altering the case of the original record even though the matching process is case insensitive. This was something that is much cleaner w/regex